.github/workflows/e2e-tests.yml

```diff
@@ -340,14 +340,22 @@ jobs:
         if: needs.conditions.outputs.should-run == 'true'
         run: yarn tronb:vite:dev
 
-      - name: Install good sed
-        if: startsWith(matrix.os, 'macos')
```

Comment:
This was actually the wrong syntax since "macos" is not at the start of the name in the matrix. This step has never been running and seems unused, I removed it.